src/assets/js/jediUtils.js

```diff
@@ -0,0 +1,50 @@
+// Special StarWWars Functions
+/**
+ * @param {string} url Url to filter a part
+ * @param {string} apiURL
+ * @returns Filtered Part
+ */
+const getCategory = (url, apiURL) => {
+  let element = url.replace(apiURL, "");
+  return element.slice(0, element.indexOf("/"));
+};
+
+/**
+ *
+ * @param {string} url
+ * @param {string} apiURL
+ * @param {*} response
+ * @returns Name or Title
+ */
+const loadNameOrTitle = (url, apiURL, response) => {
+  const item = response[getCategory(url, apiURL)].data.find(
+    (element) => element.url == url
+  );
+  return item.name || item.title || "Not defined";
+};
+/**
+ * @param {string} value Text to check if it is no number and contains https
+ * @returns if true returns true
+ */
```

Comment:
Das ist alles a bisserl verwirrend (`load...` impliziert, da wird was aus einer DB gefetched bzw via API-call, `url` vs `apiUrl` wird nicht ganz klar, desgleichen `response` etc.) - das würd ich optimieren - versuch in einem ersten Schritt das Naming der Funktionen und deren Parameter zu verbessern, so dass jedem auf den ersten Blick klar wird, was da eigentlich mit wem bzw, wozu geschieht...